backend/average-payments.php

```diff
@@ -69,7 +65,7 @@ function LastError($db) {
     }
 
     ////    Update
-    $query  = 'UPDATE `AveragePayments` SET `Total` = `Total` + \''.$amount.'\', `Count` = `Count` + 1, `Average` = ((`Total` + \''.$amount.'\') / (`Count` + 1)) WHERE `OS`=\''.$OS.'\' OR `OS`=\'total\';';
+    $query  = 'UPDATE `AveragePayments` SET `Total` = `Total` + \''.$amount.'\', `Count` = `Count` + 1, `Average` = ((`Total` + \''.$amount.'\') / (`Count` + 1)) WHERE `OS`=\''.$os.'\' OR `OS`=\'total\';';
```

Comment:
using a POST variable directly in a SQL statement is a really bad idea. we should be using `mysql_real_escape_string` on all the inputs for security